pkg/types/ccipocr3/generic_types.go

```diff
@@ -129,6 +129,8 @@ type Message struct {
 	FeeValueJuels BigInt `json:"feeValueJuels"`
 	// TokenAmounts is the array of tokens and amounts to transfer.
 	TokenAmounts []RampTokenAmount `json:"tokenAmounts"`
+
+	PseudoDeleted bool `json:"pseudoDeleted"`
```

Comment:
@ilija42 does the codec consider `json:"-"` or `mapstructure:"-"` tags?

Reply:
But this structure already have different fields like the hash inside the `RampMessageHeader`, I can add an extra map outside but I think this might work fine.

Reply:
Hmm. Does our hasher and/or encoder specifically skip over these? This might be a bug.

Reply:
I don't think so, you can use drop modifiers if you want to ignore some fields